### PR TITLE
245 Update deepgrow example with new API (part 3) 

### LINKED ISSUE #245: Update all the examples & notebooks to use new APIs
The feature comes from: https://github.com/Project-MONAI/MONAI/issues/2402

### PR DESCRIPTION
**Pull request by yiheng-wang-nv**
Signed-off-by: Yiheng Wang <vennw@nvidia.com>

Partially fixes #245 .

### Status
**Ready**

### Checks

- [ ] Notebook runs automatically `./runner [-p <regex_pattern>]`

### CODE REVIEW

**Comment by yiheng-wang-nv** [comment]
Hi @SachidanandAlle , I updated Deepgrow example, and all the code has been run successfully locally. Could you please help to review it? Thanks!

File: deepgrow/ignite/train.py [hunk 1]
```diff
@@ -221,8 +221,8 @@ def create_trainer(args):
     train_handlers = [
         LrScheduleHandler(lr_scheduler=lr_scheduler, print_lr=True),
         ValidationHandler(validator=evaluator, interval=args.val_freq, epoch_level=True),
-        StatsHandler(tag_name="train_loss", output_transform=lambda x: x["loss"]),
-        TensorBoardStatsHandler(log_dir=args.output, tag_name="train_loss", output_transform=lambda x: x["loss"]),
+        StatsHandler(tag_name="train_loss", output_transform=from_engine(["loss"], first=True)),
```
Comment (SachidanandAlle) [inline deepgrow/ignite/train.py:224]: are you sure u need this first=True flag?

**Review by SachidanandAlle — APPROVED** [review]
LGTM

File: deepgrow/ignite/train.py (hunk 1 above)
Comment (yiheng-wang-nv) [reply deepgrow/ignite/train.py:224]: Yes, according to the latest API, in order to `decollate` the `batch` and `output` (https://github.com/Project-MONAI/MONAI/blob/dev/monai/engines/workflow.py#L184), the scalar values will be added the batch dimension via duplicating (https://github.com/Project-MONAI/MONAI/blob/8365443ababac313340467e5987c7babe2b5b86a/monai/data/utils.py#L373).

**Review by Nic-Ma — APPROVED** [review]

**Pull request merged**